vipps-login-api.md

```diff
@@ -1209,33 +1209,46 @@ to be taken to the client's web page to finalise the flow.
       "interval": 5
     }
     ```
-2. The user confirms the login and is then redirected to the `redirect_uri` passed in the initial request 1. The redirect will contain `code` and `state` parameters: `{redirect_uri}?code={code}&state={state}&scope={scopes}`.
+2. The user confirms the login and is then redirected to the `redirect_uri` passed in the initial request 1. The redirect will contain a `code`: `{redirect_uri}?code={code}`.
 
-3. The merchant uses the code-parameter to obtain the login token. POST `{token_endpoint}` with `code={code}`, `grant_type=authorization_code`, and `redirect_uri={redirect_uri}` in the `application/x-www-form-urlencoded-body`. This returns (amongst others) an `access_token` that can be used to fetch userinfo.
-
-    The `code_verifier` used must correspond to the `code_challenge` used in step 1. For details see [token endpoint](#oauth-20-token) and [code_challenge](#the-code_challenge-parameter-required)
+3. The merchant uses the code-parameter to obtain the login token. POST `{token_endpoint}` with `code={code}`, `grant_type=urn:vipps:params:gran-type:ciba-redirect` in the `application/x-www-form-urlencoded-body`. This returns a ID token and an access token that can be used to fetch userinfo. The ID token is an JWS that must be validated, see https://github.com/vippsas/vipps-login-api/blob/master/vipps-login-api.md#id-token. The merchant **must validate** that it contains a `auth_req_id` they have previously received from step 2.
```

Comment:
Shouldn't it be .."the id token is a JWT with a JWS that must be validated"..

Reply:
Pretty sure JWS is the correct term for the whole payload even according to the standard itself.